content/doc/book/installing/_kubernetes.adoc

```diff
@@ -0,0 +1,524 @@
+
+*We will create a service account called jenkins:*
+
+. Paste the following snippet into a file called jenkins-sa.yaml:
++
+[source,bash]
+----
+---
+  apiVersion: v1
+  kind: ServiceAccount
+  metadata:
+    name: jenkins
+---
+  apiVersion: rbac.authorization.k8s.io/v1
+  kind: ClusterRole
+  metadata:
+    annotations:
+      rbac.authorization.kubernetes.io/autoupdate: "true"
+    labels:
+      kubernetes.io/bootstrapping: rbac-defaults
+    name: jenkins
+  rules:
+  - apiGroups:
+    - '*'
+    resources:
+    - statefulsets
+    - services
+    - replicationcontrollers
+    - replicasets
+    - podtemplates
+    - podsecuritypolicies
+    - pods
+    - pods/log
+    - pods/exec
+    - podpreset
+    - poddisruptionbudget
+    - persistentvolumes
+    - persistentvolumeclaims
+    - jobs
+    - endpoints
+    - deployments
+    - deployments/scale
+    - daemonsets 
+    - cronjobs
+    - configmaps
+    - namespaces
+    - events
+    - secrets
+    verbs:
+    - create
+    - get
+    - watch
+    - delete
+    - list
+    - patch
+    - apply
+    - update
+    - apiGroups:
+    - ""
+    resources:
+    - nodes
+    verbs:
+    - get
+    - list
+    - watch
+    - update
+---
+  apiVersion: rbac.authorization.k8s.io/v1
+  kind: ClusterRoleBinding
+  metadata:
+    annotations:
+      rbac.authorization.kubernetes.io/autoupdate: "true"
+    labels:
+kubernetes.io/bootstrapping: rbac-defaults
+    name: jenkins
+  roleRef:
+    apiGroup: rbac.authorization.k8s.io
+    kind: ClusterRole
+    name: jenkins
+  subjects:
+  - apiGroup: rbac.authorization.k8s.io
+    kind: Group
+    name: system:serviceaccounts:jenkins
+----
++
+A ClusterRole is a set of permissions that can be assigned to resources within a given cluster.
+Kubernetes APIs are categorized into API groups, based on the API objects that they relate to.
+While creating a ClusterRole, you can specify the operations that can be performed by the ClusterRole on one or more API objects in one or more API groups, just as we have done above.
+ClusterRoles have several uses. You can use a ClusterRole to:
+
+* define permissions on namespaced resources and be granted within individual namespace(s)
+* define permissions on namespaced resources and be granted across all namespaces
+* define permissions on cluster-scoped resources
++
+If you want to define a role within a namespace, use a Role; if you want to define a role cluster-wide, use a ClusterRole.
++
+On the other hand, a role binding grants the permissions defined in a role to a user or set of users. It holds a list of subjects (users, groups, or service accounts), and a reference to the role being granted. A RoleBinding grants permissions within a specific namespace whereas a ClusterRoleBinding grants that access cluster-wide.
```

Comment:
Stylistically, it would be nice to add line breaks after each sentence.  That way, if changes are made, it's easier to see what changed when.

Reply:
Also it feels like this and the section below it are related.  Maybe take the concepts and combine them?